src/NServiceBus.Core/Sagas/Sagas.cs

```diff
@@ -16,25 +15,20 @@ public sealed class Sagas : Feature
     /// </summary>
     public Sagas()
     {
-        Defaults(s =>
-        {
-            conventions = s.Get<Conventions>();
-
-            var sagas = s.GetAvailableTypes().Where(IsSagaType).ToList();
-            if (sagas.Count > 0)
-            {
-                conventions.AddSystemMessagesConventions(t => IsTypeATimeoutHandledByAnySaga(t, sagas));
```

Comment:
Side note; now this is no longer used in core so my hunch is that we can deprecate. It is still used in ASB and SC thought https://github.com/search?q=org%3AParticular%20AddSystemMessagesConventions&type=code